### ISSUE TITLE
Improve error message in lncli when "changepassword" is used after wallet is already unlocked

### ISSUE THREAD

**Issue by molxyz** [lightningnetwork/lnd#2529]
### Background
When LND node is unlocked and running, try to change password by entering on a different terminal screen. LND seems to allow the password to be changed but actually it is not changed. 
### Your environment
* version of `lnd`
"version": "0.5.1-beta commit=v0.5.1-beta-461-g375be936ce3df375e6c2686b190563d7076ecf40",
* which operating system (`uname -a` on *Nix)
Ubuntu 16.04
Bitcoind v0.17.99
### Steps to reproduce
While lnd is already unlocked and running, on another terminal screen try to change your password.
 ```
~/.lnd$ tlncli changepassword
Input current wallet password:
Input new wallet password:
Confirm new wallet password:
[lncli] Wallet is encrypted. Please unlock using 'lncli unlock', or set password using 'lncli create' if this is the first time starting lnd.
```
Now try to unlock the wallet when it's already unlocked... lol
```
~/.lnd$ tlncli unlock
Input wallet password:
[lncli] Wallet is already unlocked
```
Then stop LND from running. Now try to restart LND with the new password.
```
~/.lnd$ tlncli unlock
Input wallet password:
[lncli] rpc error: code = Unknown desc = invalid passphrase for master public key
```

### Expected behaviour
If trying to change password when the node is already unlocked and running, LND should not allow the steps to happen  and should issue an error message such as 
```
Cannot change password while wallet is already unlocked. Please stop lnd and restart to change your password.
```

**Comment by Roasbeef** [comment]
You can only run `changepassword` with the wallet unlocker active. The command tries, but doesn't actually succeed. We should update the error message there to make this more explicit.

**Comment by stridentbean** [comment]
If I'm reading this correctly, the wallet unlocker service is deactivated after the wallet is unlocked. Is there any way to activate the service on the fly? Right now, I have to restart lnd and then change the password.

**Comment by wpaulino** [comment]
> Is there any way to activate the service on the fly?

Not at the moment.

**Comment by redek-zelton** [comment]
may be can help https://github.com/lightningnetwork/lnd/blob/master/docs/wallet.md

* Changing the wallet password is possible but only while the wallet is locked. So after restarting lnd, instead of using the unlock command, the changepassword command can be used:
```bash
$ lncli changepassword
# or 
$lncli --network=testnet changepassword
# or
$tlncli changepassword
```
* This will ask for the old/existing password and a new one. If successful, the database is re-encrypted with the new password and then the wallet is also unlocked in the process.

**Comment by redek-zelton** [comment]
here: https://api.lightning.community/?shell#v1-changepassword
$ MACAROON_HEADER="Grpc-Metadata-macaroon: $(xxd -ps -u -c 1000 $LND_DIR/data/chain/bitcoin/simnet/admin.macaroon)"
$ curl -X POST --cacert $LND_DIR/tls.cert --header "$MACAROON_HEADER" https://localhost:8080/v1/changepassword  \
    -d '{ "current_password":<byte>,"new_password":<byte>,"stateless_init":<boolean>,"new_macaroon_root_key":<boolean>, }' 
{ 
    "admin_macaroon": <byte>, 
}

**Comment by guggero** [comment]
This issue is only about the error message, really.

Currently if you try to change the password after the wallet is already unlocked, `lncli` will return:
```
[lncli] Wallet is encrypted. Please unlock using 'lncli unlock', or set password using 'lncli create' if this is the first time starting lnd.
```

While it should probably say something more specific or more informative.

**Comment by guggero** [comment]
Fixed by #6349.
